src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -95,10 +95,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + feedbackResponseComment.feedbackSessionName + "<br>"
                            + "by: " + feedbackResponseComment.giverEmail + "<br>"
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
+            // createdAt, lastEditedAt, commentGiverName and commentEditorName are required to generate editedCommentDetails
```

Comment:
As the above comment exceeds 125 words limit, will it be okay if I use `/* */` for comment.Just asking because I don't see this used in the code.